_extras/training_calendar.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: "Instructor Training Calendar for Carpentries Partners"
+permalink: /training_calendar/
+---
+
+
+#### Carpentries Instructor Training Calendar
+###  September - December 2017
+
+
+If you are a partner site ready to schedule your instructor training session, please contact maneesha@carpentries.org with your first and second choices for dates.
+
+All trainings are offered online from 9 am to 5 pm in the time zone noted.  You do not need to be in the same time zone yourself but need to accommodate that time zone. 
+
+Please keep in mind Daylight savings time changes on November 5, 2017.
+
+# United States
+
+##### Eastern time
+* September 21-22, 2017
+* October 24-25, 2017
+* November 14-15, 2017
+
+##### Central time
+* October 5-6, 2017
+* November 2-3, 2017
+
+##### Mountain time
+* October 30-31, 2017
+* December 12-13, 2017
+
+#### Pacific time
+* September 25-26, 2017
+* October 18-19, 2017
+* November 15-16, 2017
+
+# United Kingdom
+* Training dates to be announced
+
+# Europe
+
+##### Central European time
+* October 30-31, 2017
+* December 4-5, 2017
+
+
+
+
```

Comment:
Trainings listed here are ones that partner sites can sign up for.   We are not listing AU/NZ trainings here because those are being scheduled for specific sites, not for any site to sign up for.